### PR TITLE
update dev900 7.2

### PR DESCRIPTION
**Pull request by liang-csc**
pls check

### CODE REVIEW

File: test.txt [hunk 1]
```diff
@@ -14,7 +14,7 @@ changes 1
 Added on Editor
 Branch dev900
 Author dev900
-changes 7.1
+changes 7.2
```
Comment (lianglinzu) [inline test.txt:17]: test

Comment (liang-csc) [reply test.txt:17]: ok?

Comment (lianglinzu) [reply test.txt:17]: of course

**Comment by liang-csc** [comment]
????

File: test.txt (hunk 1 above)
Comment (lianglinzu) [reply test.txt:17]: test

Comment (liang-csc) [reply test.txt:17]: hi

Comment (lianglinzu) [reply test.txt:17]: hi

**Pull request merged**